.changeset/curvy-vans-worry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@layerzerolabs/oft-solana-example": patch
```

Comment:
minor or patch? or doesn't matter?

Reply:
I didn't even realize I had put `patch`, sorry you are right.